File: 프로그래머스/1/12925. 문자열을 정수로 바꾸기/문자열을 정수로 바꾸기.py
def solution(s):
    answer = 0
    if(s[0] == '-'):
        for i in range(1, len(s)):
            answer += int(s[i])
            answer *= 10
        answer *= -1
    elif(s[0] == '+'):
        for i in range(1, len(s)):
            answer += int(s[i])
            answer *= 10
    elif(s[0] >= '0') & (s[0] <= '9'):
        for i in range(0, len(s)):
            answer += int(s[i])
            answer *= 10
    answer //= 10
    return answer

File: 프로그래머스/1/12925. 문자열을 정수로 바꾸기/문자열을 정수로 바꾸기.swift
func solution(_ s:String) -> Int {
    var result : Int = 0
    var bool : Bool = false
    var string : String = ""
    for stringInt in s {
        if stringInt == "-" {
            bool = true
        }else{
            string += String(stringInt)
        }
    }
    result = Int(string) ?? 0
    if bool == true{
        result = -result
    }
    return result
}